instances/patchClampVariation/cellAttachedPatch.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+    "@context": {
```

Comment:
these is a technique... I do not think that we should capture it again. I do see why you only want to separate the patch clamp techniques but we should find a better way for this.

Any ideas?

Reply:
this applies for patchClampVariations you add. Other way around, but possible rejected by the EBRAINS curation team: reducing technique to "patch clamp" and leaving all variations in here. Does this work? I can find a solution for the curation team (e.g. by allowing not only technique but also the patch variations) for technique properties.

Reply:
We could add them as techniques, but then how do we limit the field to only a specific subset of instances?

Reply:
exactly for this we do not have yet any functionality...

I think we should do it the other way around. I will delete the more specific patch clamp techniques we have from the techniques terminology and we leave them in patchClampVariations. For the technique properties I will add then the patchClampVariations as additional linkedType so that users can immediately state that they are doing "whole cell patch clamp". does this make sense?

Reply:
perfect :) this issue might occur for future extensions (and other parts of ephys) as well.